src/utils/dom/querySelectorAll.js

```diff
@@ -12,5 +12,22 @@ governing permissions and limitations under the License.
 
 import toArray from "../toArray";
 
-export default (context, selector) =>
-  toArray(context.querySelectorAll(selector));
+const SIBLING_PATTERN = /^\s*>/;
+
+export default (context, selector) => {
+  if (!SIBLING_PATTERN.test(selector)) {
+    return toArray(context.querySelectorAll(selector));
+  }
+
+  const tag = `alloy${+new Date()}`;
```

Comment:
Although equivalent, it is more clear to change this to `alloy${new Date().getTime()}`.

Reply:
@jonsnyder I have replaced it with `Date.now()` as far as I saw it is supported in IE 9+ and all other modern browsers.